sale_invoice_enduser/tests/__init__.py

```diff
@@ -0,0 +1 @@
+from . import test_sale_order
```

Comment:
@YogeshMahera-SerpentCS Please do not include irrelevant update here. Why don't you double check the entire content of PR yourself (this kind of mistake should be easy to spot) before asking for a review?

Reply:
Thanks, @yostashiro for the check.